_rules/meta-viewport-b4f0c3.md

```diff
@@ -24,22 +24,37 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
+This rule applies to each `content` attribute for which all of the following are true:
 
-- the `content` [attribute value][] has the `user-scalable` property; or
-- the `content` [attribute value][] has the `maximum-scale` property.
+- the attribute is defined on a `meta` element with a `name` [attribute value][] of `viewport`; and
+- the [attribute value][] has at least one of the `maximum-scale` or `user-scalable` keys.
+
+For the purpose of this rule, the [attribute value][] of the `content` attribute is a list of key/value pairs.
 
 ## Expectation 1
 
-For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `no`.
+For each test target's [attribute value][], at least one of the following is true:
+
+- the `user-scalable` key is not defined; or
+- the `user-scalable` key is paired with a value of `yes`, `device-width` or `device-height`; or
+- the `user-scalable` key is paired with a numeric value which is not in the range ]-1;1[.
```

Comment:
I would prefer we simplify this a bit. I don't think this is any less ambiguous than what you had:

```suggestion
- `user-scalable` is not defined; or
- `user-scalable` is `yes`, `device-width` or `device-height`; or
- `user-scalable` is a number which is not between -1 and 1.
```